README.md

```diff
@@ -34,95 +37,93 @@ See [update build process svg](build-process-mermaid.md)
 ### Getting started - prerequisites
 ---
 
-You can use `podman` or `docker`. These examples use `podman`. Note
-that we support running in a privileged or unprivileged mode (detailed
-below). Regardless of whether you are using privileged or unprivileged
-mode you'll need access to `/dev/kvm` as the build process runs a
-virtual machine in order to generate the target image. If you're running
-this in a VM, you must enable [nested virt](https://docs.fedoraproject.org/en-US/quick-docs/using-nested-virtualization-in-kvm/).
-There are various public cloud options that provide bare metal, such as [Packet](https://www.packet.com/), [GCE nested virt](https://cloud.google.com/compute/docs/instances/enable-nested-virtualization-vm-instances), EC2 `i3.metal` instances, [IBM Bare Metal](https://www.ibm.com/cloud/bare-metal-servers), etc.
+The default instructions here use `podman`, but it is also possible to use
+`docker`.
```

Comment:
i wonder how many of our developers are using docker. Maybe we should try to poll around and make sure we aren't going to disrupt anyone. Maybe andrew still uses docker vs podman?? Not sure if anyone else would or not.